diyepw/meteorology.py

```diff
@@ -177,8 +177,8 @@ def write_epw(self, save_path):
                 'https://www1.ncdc.noaa.gov/pub/data/noaa/isd-lite/ for an actual meteorological year (AMY)',
 
                 'DATA PERIODS,1,1,Data,' + first_day_of_week + ', 1/1, 12/31'
-            ]) + os.linesep)
-            epw_file.write(self._observations.to_csv(header=False, index=False))
+            ])
+            self._observations.to_csv(epw_file, header=False, index=False)
```

Comment:
This is more of a code preference, using the filehandle as an argument of the `DataFrame.to_csv` method. In theory, you don't need to generate the string in memory before writing to the file...